package.json

```diff
@@ -13,54 +13,55 @@
     "test": "vitest run"
   },
   "dependencies": {
-    "@cowprotocol/cow-sdk": "^2.1.0",
+    "@cowprotocol/cow-sdk": "2.1.0",
```

Comment:
I was having some issues with this package with versions other than this one, so pinning it